### PR TITLE
TDR-2522 - Show metadata summary page based on metadata type

### PR DESCRIPTION
**Pull request by vimleshtna**
Show metadata summary page based on metadata type
- Use display properties to show the metadata values

### CODE REVIEW

File: test/testUtils/FrontEndTestHelper.scala [hunk 1]
```diff
@@ -397,10 +396,70 @@ trait FrontEndTestHelper extends PlaySpec with MockitoSugar with Injecting with
...
+            dp.DisplayProperties.Attributes("Datatype", Some("boolean"), Text),
+            dp.DisplayProperties.Attributes("ComponentType", Some("radial"), Text),
+            dp.DisplayProperties.Attributes("Description", Some("Is the title closed?"), Text),
+            dp.DisplayProperties.Attributes("Name", Some("Is the title closed?"), Text),
+            dp.DisplayProperties.Attributes("Editable", Some("true"), Boolean),
+            dp.DisplayProperties.Attributes("Label", Some("Yes|No, this title can be made public"), Text),
+            dp.DisplayProperties.Attributes("Group", Some("2"), Text),
+            dp.DisplayProperties.Attributes("MultiValue", Some("false"), Boolean),
+            dp.DisplayProperties.Attributes("Ordinal", Some("25"), Integer),
+            dp.DisplayProperties.Attributes("UnitYpe", Some("Years"), Integer),
```
Comment (TomJKing) [inline test/testUtils/FrontEndTestHelper.scala:462]: Attribute name should be `UnitType`

**Review by TomJKing — COMMENTED** [review]
Looks good. One minor comment. Thanks

File: test/testUtils/FrontEndTestHelper.scala (hunk 1 above)
Comment (vimleshtna) [reply test/testUtils/FrontEndTestHelper.scala]: Good spot. Actually, it's not required for TitleClosed property. I removed it.

**Review by TomJKing — APPROVED** [review]
Looks good. Thanks

**Pull request merged**